code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -156,6 +156,20 @@
 
 /obj/item/clothing/accessory/stethoscope/attack(mob/living/carbon/human/M, mob/living/user)
 	if(ishuman(M) && isliving(user))
```

Comment:
Prevents shadekin from using it. What even is the point!!!

Reply:
On line 172 in your current edit, just print the belly messages you found so far and then return, if they are not a human. Everything below that is suitable only for humans. So change to isliving() at the top and then a small block for:
```
if(!ishuman(M))
    //do the message to whoever
    //return
```